impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -149,11 +228,25 @@ AllocatorVK::AllocatorVK(std::weak_ptr<Context> context,
     VALIDATION_LOG << "Could not create memory allocator";
     return;
   }
+
+  {
+    // Query texture support.
+    // TODO(jonahwilliams):
+    // https://github.com/flutter/flutter/issues/129784
+    physical_device.getMemoryProperties(&memory_properties_);
+
+    for (auto i = 0u; i < memory_properties_.memoryTypeCount; i++) {
+      if (memory_properties_.memoryTypes[i].propertyFlags &
+          vk::MemoryPropertyFlagBits::eLazilyAllocated) {
+        supports_memoryless_textures_ = true;
```

Comment:
`break` after since it doesn't seem relevant to iterate through once we find one?

Reply:
Done